run/panic.go

```diff
@@ -13,5 +20,5 @@ func HandlePanics() {
 	if obj == nil {
 		return
 	}
-	panicHandler <- obj
+	panicHandler <- PanicPayload{obj, string(debug.Stack())}
```

Comment:
Not sure what information we propagate with `obj` honestly, but `debug.Stack()` at this point is lost, unless we forward it.